app/webpacker/components/Persons/Person/Results.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { useMemo, useState } from 'react';
+import { Divider, Table } from 'semantic-ui-react';
+import _ from 'lodash';
+import { events, roundTypes } from '../../../lib/wca-data.js.erb';
+import { EventSelector } from '../../wca/EventSelector';
+import { competitionUrl } from '../../../lib/requests/routes.js.erb';
+import I18n from '../../../lib/i18n';
+import { formatAttemptResult } from '../../../lib/wca-live/attempts';
+
+const colorForResult = (regionalRecord, pbMarker) => {
```

Comment:
I think the `let` could be removed and each `recordColor = ...` assignment could be replaced with `return ...`, if the order is reversed - i.e. regional record then pb marker then empty case.

The name should be `styleForResult`, or the function refactored to just return the string of the color.